plugins/tech-radar/src/components/RadarGrid/RadarGrid.tsx

```diff
@@ -40,6 +40,7 @@ const useStyles = makeStyles<Theme>(theme => ({
     fill: theme.palette.text.primary,
     fontSize: '25px',
     fontWeight: 800,
+    opacity: 0.35,
```

Comment:
This is very opaque, I wonder if it could be problematic for people of poor eyesight. Let's try it and see if we get feedback on it.

On that note, while we're here, it might be nice to add a name to this style so it can be overridden, while we're modifying this file anyway. Similar to https://github.com/backstage/backstage/blob/9d9348b972e91a253308ce11a063897abefe1255/packages/core-components/src/components/Lifecycle/Lifecycle.tsx#L43 for example. good naming choice: `PluginTechRadarGrid` perhaps?

Reply:
Good catch, I can bump it up to .7 and it still gives a similar visual effect as the Zalando radar but has a contrast > 3.